### DISCUSSION TITLE
Using Sequelize

### DISCUSSION THREAD

**Discussion by younes-alouani** [nextauthjs/next-auth#4333] [Help]
### Question 💬

I tried to use Sequelize as an adapter in a next-auth but i faced problems creating the config file and building the models files. 
Can I find any tutorial or detailed documentation ? 

### How to reproduce ☕️

-

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by ThangHuuVu** [comment]
Can you provide more about the problem you're facing? 
In case you haven't seen it, here's the doc for Sequelize adapter: https://next-auth.js.org/adapters/sequelize

**Reply by younes-alouani** [reply]
Hello, 
The guide about sequelize is not complete, for someone who is a novice, following the steps indicated, he will not be able to make the solution work. 
Indeed, several elements are missing, namely:
1- The need to install custom packages adapted to the database used. For Postgresql, for example, you need to install pg, pg-hstore
2- To create tables, make migrations and interact with the database, you need to install the package: sequelize-cli
3- To configure sequelize, you need to create a .sequelizerc file to indicate the path of the config.js file (instead of config.json created by default) 
4- At the config.js level, it is possible to call environment variables via process.env . However, it will be necessary to install the package dotenv and require it in the config file

**Comment by adrianbucks** [comment]
Any luck finding working examples?
I've set sequelize as adapter and it creates the default tables in development but it doesn't pick new models defined in the options object of adapter (client, options).

Looking at /models/index file it seems that it defines only the 4 default tables
